frontend_v2/src/app/services/global.service.ts

```diff
@@ -439,7 +439,7 @@ export class GlobalService {
     if (err.status === 401) {
       this.checkTokenValidity(err, toast);
     } else if (err.status === 403 && toast) {
-      this.showToast('error', err.error['error'], 5);
+      this.showToast('error', err.error['detail'], 5);
```

Comment:
@ankitpal1029 we should check if `error` key exists in the error response. If it does, then we show that in the toast. If not, then we look for `detail` key and show that